Tasks/Src/Task_CheckContactor.c

```diff
@@ -1,16 +1,56 @@
 /* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
-#include "RTOS_BPS.h"
+#include "config.h"
 #include "Contactor.h"
+#include "CANbus.h"
 #include "Tasks.h"
+#include "BSP_UART.h"
+#include "Voltage.h"
+#include "CAN_Queue.h"
 
-/* 
- * Launch this task once the contactor has been closed.
- * This will trip the BPS if the contactor is open.
- * The purpose of this is to trip the BPS if the emergency
- * cutoff switch is pressed.
- */
 void Task_CheckContactor(void *p_arg) {
     (void)p_arg;
+    
+    CANMSG_t CANMSG;
+    CANPayload_t CanPayload; 
+    CanPayload.idx    = 0;
+    CanPayload.data.b = 1;
+
+    // If a contactor is on before we turn it on in this task, it may have failed and welded shut
+    if(Contactor_GetState(HVHIGH_CONTACTOR) || Contactor_GetState(HVLOW_CONTACTOR)) {
```

Comment:
keep consistent spacing
```suggestion
    if (Contactor_GetState(HVHIGH_CONTACTOR) || Contactor_GetState(HVLOW_CONTACTOR)) {
```